packages/account/src/Components/poi/poi-form-on-signup/idv-doc-submit-on-signup/idv-doc-submit-on-signup.tsx

```diff
@@ -102,7 +102,12 @@ export const IdvDocSubmitOnSignup = ({
                 <Form className='proof-of-identity__container proof-of-identity__container--reset mt5-layout'>
                     <section className='mt5-layout__container'>
                         <FormSubHeader title={localize('Identity verification')} />
-                        <IDVForm hide_hint={false} selected_country={citizen_data} class_name='idv-layout' />
+                        <IDVForm
+                            hide_hint={false}
```

Comment:
*NIT*: `hide_hint` prop can be ignored

Reply:
omg 😨  I thought I removed it :)
thanks man. it might’ve  been other place :)